common/manalink.ts

```diff
@@ -0,0 +1,27 @@
+export type Manalink = {
+  // The link to send: https://manifold.markets/send/{slug}
+  // Also functions as the unique identifier for the link.
+  slug: string
+
+  // Note: we assume both fromId and toId are of SourceType 'USER'
+  fromId: string
```

Comment:
How would we cross reference txns to mana links? Maybe the txn will have the id / slug of the manalink in a data field.